standard/expressions.md

```diff
@@ -1279,11 +1279,16 @@ In a member access of the form `E.I`, if `E` is a single identifier, and if the
 
 ### §null-conditional-member-access Null Conditional Member Access
 
-A *null_conditional_member_access* consists of a *primary_expression* followed by a "`?.`" token, followed by an *Identifier*, optionally followed by a *type_argument_list*.
+A *null_conditional_member_access* consists of a *primary_expression* followed by a "`?.`" token, followed by an *Identifier* with an optional *type_argument_list*, followed by zero or more *unconditional_access_part*s.
 
 ```ANTLR
 null_conditional_member_access
-    : primary_expression '?.' Identifier type_argument_list?
+    : primary_expression '?.' Identifier type_argument_list? unconditional_access_part*
```

Comment:
"unconditional_access_part" sounds like something that will be evaluated unconditionally, and I don't *think* that's what we're looking for, is it?

Reply:
I agree its not a great name, but probably better than my other thought of `null_faulting_access`, over to the wordsmiths!

Reply:
`?.` is a pair of tokens.

Reply:
Changed